### PR TITLE
Remove more jcenter references from the docs

### PR DESCRIPTION
**Pull request by rieske**
https://github.com/gradle/gradle/issues/16035

This should only leave the `declaring_repositories` page talking about jcenter, which I will handle separately.

### CODE REVIEW

File: subprojects/docs/src/snippets/plugins/buildscript/groovy/build.gradle [hunk 1]
```diff
@@ -1,7 +1,9 @@
 // tag::buildscript_block[]
 buildscript {
     repositories {
-        jcenter()
+        maven {
+            url "https://plugins.gradle.org/m2/"
+        }
```
Comment (bmuskalla) [inline subprojects/docs/src/snippets/plugins/buildscript/groovy/build.gradle:6]:
```suggestion
        gradlePluginPortal()
```

File: subprojects/docs/src/snippets/plugins/buildscript/kotlin/build.gradle.kts [hunk 2]
```diff
@@ -1,7 +1,9 @@
 // tag::buildscript_block[]
 buildscript {
     repositories {
-        jcenter()
+        maven {
+            url = uri("https://plugins.gradle.org/m2/")
+        }
```
Comment (bmuskalla) [inline subprojects/docs/src/snippets/plugins/buildscript/kotlin/build.gradle.kts:6]:
```suggestion
        gradlePluginPortal()
```

**Review by bmuskalla — COMMENTED** [review]
Question inline wrt using the proper accessor

**Comment by donat** [comment]
👀

**Pull request merged**